com.sap.cloud.lm.sl.persistence/src/main/java/com/sap/cloud/lm/sl/persistence/services/FileSystemFileService.java

```diff
@@ -140,6 +145,30 @@ private Path getFilesPerSpaceDirectory(String space) {
         return invalidateEntriesWithoutContent(allEntries);
     }
 
+    @Override
+    public List<FileEntry> listByModificationTime(final Date modificationTime) throws FileStorageException {
+        List<FileEntry> fileEntriesAttributes = super.listByModificationTime(modificationTime);
+        invalidateEntriesWithoutContent(fileEntriesAttributes);
+
+        final List<FileEntry> oldEntries = new ArrayList<FileEntry>();

```

Comment:
oldEntries -> result?

Reply:
Will remain in that way. Other functions have the same name convention.